<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="5" passed="4" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[Method 2 is running]]>
    </line>
    <line>
      <![CDATA[Method 3 is running]]>
    </line>
    <line>
      <![CDATA[Method 4 is running]]>
    </line>
    <line>
      <![CDATA[Method 5 is running]]>
    </line>
  </reporter-output>
  <suite started-at="2022-04-06T16:25:07 IST" name="Default suite" finished-at="2022-04-06T16:25:07 IST" duration-ms="268">
    <groups>
    </groups>
    <test started-at="2022-04-06T16:25:07 IST" name="Default test" finished-at="2022-04-06T16:25:07 IST" duration-ms="268">
      <class name="failedXML_Study.Failed_XML1">
        <test-method signature="Method1()[pri:0, instance:failedXML_Study.Failed_XML1@6356695f]" started-at="2022-04-06T16:25:07 IST" name="Method1" finished-at="2022-04-06T16:25:07 IST" duration-ms="134" status="FAIL">
          <exception class="org.testng.internal.thread.ThreadTimeoutException">
            <message>
              <![CDATA[Method failedXML_Study.Failed_XML1.Method1() didn't finish within the time-out 100]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method failedXML_Study.Failed_XML1.Method1() didn't finish within the time-out 100
at org.testng.internal.MethodInvocationHelper.invokeWithTimeoutWithNewExecutor(MethodInvocationHelper.java:384)
at org.testng.internal.MethodInvocationHelper.invokeWithTimeout(MethodInvocationHelper.java:283)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:604)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.internal.thread.ThreadTimeoutException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Method1 -->
        <test-method signature="Method2()[pri:0, instance:failedXML_Study.Failed_XML1@6356695f]" started-at="2022-04-06T16:25:07 IST" name="Method2" finished-at="2022-04-06T16:25:07 IST" duration-ms="7" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Method 2 is running]]>
            </line>
          </reporter-output>
        </test-method> <!-- Method2 -->
        <test-method signature="Method3()[pri:0, instance:failedXML_Study.Failed_XML1@6356695f]" started-at="2022-04-06T16:25:07 IST" name="Method3" finished-at="2022-04-06T16:25:07 IST" duration-ms="5" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Method 3 is running]]>
            </line>
          </reporter-output>
        </test-method> <!-- Method3 -->
        <test-method signature="Method4()[pri:0, instance:failedXML_Study.Failed_XML1@6356695f]" started-at="2022-04-06T16:25:07 IST" name="Method4" finished-at="2022-04-06T16:25:07 IST" duration-ms="4" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Method 4 is running]]>
            </line>
          </reporter-output>
        </test-method> <!-- Method4 -->
        <test-method signature="Method5()[pri:0, instance:failedXML_Study.Failed_XML1@6356695f]" started-at="2022-04-06T16:25:07 IST" name="Method5" finished-at="2022-04-06T16:25:07 IST" duration-ms="3" status="PASS">
          <reporter-output>
            <line>
              <![CDATA[Method 5 is running]]>
            </line>
          </reporter-output>
        </test-method> <!-- Method5 -->
      </class> <!-- failedXML_Study.Failed_XML1 -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
